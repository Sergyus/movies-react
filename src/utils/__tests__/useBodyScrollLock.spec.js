import React from 'react';
import { create, act } from 'react-test-renderer';
import useBodyScrollLock from '../useBodyScrollLock';

describe('Body scroll lock work correctly', () => {
  const Component = () => {
    useBodyScrollLock();
    return <h1>Hello</h1>;
  };
  it('Set overflow to hidden', () => {
    let tree;
    act(() => {
      tree = create(<Component />);
    });
    expect(tree.toJSON()).toMatchSnapshot();
    tree.unmount();
  });
});
