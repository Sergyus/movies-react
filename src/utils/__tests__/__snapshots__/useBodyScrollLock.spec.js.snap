// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body scroll lock work correctly Set overflow to hidden 1`] = `
<h1>
  Hello
</h1>
`;
